crates/core_arch/src/x86/sse.rs

```diff
@@ -219,6 +220,7 @@ pub unsafe fn _mm_max_ss(a: __m128, b: __m128) -> __m128 {
 #[cfg_attr(test, assert_instr(maxps))]
 #[stable(feature = "simd_x86", since = "1.27.0")]
 pub unsafe fn _mm_max_ps(a: __m128, b: __m128) -> __m128 {
+    // See the `test_mm_min_ps` test why this can't be implemented using `simd_fmax`.
```

Comment:
I think it would be better to add similar tests to the other intrinsics using `simd_fmax` and `simd_fmin`, that check subnormals, and also that check the behavior when the first argument is nan, and the second non-nan, and viceversa.

Reply:
How do I create a subnormal? As far as I understand they are close to zero, but I don't know how close.

Reply:
> How do I create a subnormal? 

Check out the docs for `f{32,64}::is_normal()`. Each floating-point type has a `MIN_POSITIVE` number, and all numbers between that one and zero (I think in range: `(-MIN_POSITIVE, MIN_POSITIVE)`) are subnormal. I don't know if creating them from a literal returns 0.0 or not. But if they do, then checking permutations of -0.0, 0.0, and NaN should be enough, e.g., (-0.0, 0.0), (0.0, -0.0), (1.0, NaN), (NaN, 1.0).

Reply:
`0.000000000000000000000000000000000000000000001f32.is_normal()` returns false and transmuting it to `[u8; 4]` gives `[1, 0, 0, 0]`. Do you want to check permutations with that number too? Or should I just use `0.0`?

Reply:
> Do you want to check permutations with that number too? 

Yes, we should check that too :)